bgpd/bgp_labelpool.c

```diff
@@ -0,0 +1,539 @@
+/*
+ * BGP Label Pool - Manage label chunk allocations from zebra asynchronously
+ *
+ * Copyright 2018, LabN Consulting, L.L.C.
+ *
+ * This file is part of GNU Zebra.
```

Comment:
https://github.com/FRRouting/frr/blob/master/doc/developer/workflow.rst

Please use the appropriate source file header